README.md

```diff
@@ -49,6 +52,81 @@ if err != nil {
 }
 ```
 
+### Using the default config with env vars
+The default configuration makes use of the `TFE_ADDRESS` and `TFE_TOKEN` environment variables.
+
+1. `TFE_ADDRESS` - URL of a Terraform Cloud or Terraform Enterprise instance. Example: `https://tfe.local`
+1. `TFE_TOKEN` - An [API token](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/api-tokens) for the Terraform Cloud or Terraform Enterprise instance.
+
+**Note:** Alternatively, you can set `TFE_HOSTNAME` which serves as a fallback for `TFE_ADDRESS`. It will only be used if `TFE_ADDRESS` is not set and will resolve the host to an `https` scheme. Example: `tfe.local` => resolves to `https://tfe.local`
+
+The environment variables are used as a fallback to configure TFE client if the Address or Token values are not provided as in the cases below:
+
+#### When tfe.DefaultConfig is used
+```go
+import (
+  "context"
+  "log"
+
+  "github.com/hashicorp/go-tfe"
+)
+
+client, err := tfe.NewClient(tfe.DefaultConfig())
+if err != nil {
+	log.Fatal(err)
+}
+
+orgs, err := client.Organizations.List(context.Background(), nil)
+if err != nil {
+	log.Fatal(err)
+}
+```
```

Comment:
I like that the example is explicit about what configuration we're using, but it may also be worth mentioning that `NewClient()` [will call](https://github.com/hashicorp/go-tfe/blob/b4072d56973028fc00921161247d66b8bd56716b/tfe.go#L271) `DefaultConfig()` so if `nil` is passed it is effectively equivalent.

Reply:
The second example, lines#86 `When no tfe.Config is provided` captures this scenario, I am not sure where to add this additional summary in a way that is not confusing. Do you have any suggestions of where we can add this additional context?

Reply:
I'm not sure we have to provide an extra example for passing nil, we can simply add a code comment above line 75:
`// Passing nil to this method will also use the default configuration`.

Reply:
done 🎉